src/analytics/script/analytics/collector/page/page.js

```diff
@@ -19,9 +19,14 @@ class Page {
     this.referrer = clear ? '' : this._config.referrer;
     this.title = clear ? '' : this._config.title;
     this.name = clear ? '' : this._config.name;
+    this.id = clear ? '' : this._config.id;
+    this.author = clear ? '' : this._config.author;
+    this.date = clear ? '' : this._config.date;
     this._labels = clear || !this._config.labels ? ['', '', '', '', ''] : this._config.labels;
+    this._tags = clear || !this._config.tags ? ['', '', '', '', ''] : this._config.tags;
     this._categories = clear || !this._config.categories ? ['', '', ''] : this._config.categories;
     this._labels.length = 5;
+    this._tags.length = 5;
```

Comment:
`this._tags.length = 5;`
pas besoin de cette ligne qui sert à forcer la limite sur labels si jamais il y en avait plus de 5
et d'ailleurs, rapprocher la ligne labels.length de la définition de labels